Tests/ClientRuntimeTests/OrchestratorTests/OrchestratorTests.swift

```diff
@@ -493,7 +493,7 @@ class OrchestratorTests: XCTestCase {
             let b = self.traceOrchestrator(trace: partitionIdTrace)
             // We can force a getPartitionId to fail by explicitly setting the host to ''
             b.interceptors.addModifyBeforeRetryLoop({ context in
-                context.updateRequest(updated: context.getRequest().toBuilder().withHost("").build())
+                context.updateRequest(updated: try context.getRequest().toBuilder().withHost("").build())
```

Comment:
Same thing here - might be missing something but I don't think this method throws. It's in a few places